utils/scripts/ocm/README.md

```diff
@@ -0,0 +1,100 @@
+# Provision Openshift Dedicated Cluster in AWS
+Script to install Openshift Dediated (OSD) on AWS with the aid of Openshift Cluster Manager CLI tool ([ocm-cli](https://github.com/openshift-online/ocm-cli.git))
+
+# Prerequisites
+- Python 3.x
+- Install ocm cli
```

Comment:
Give a for example link to one of the latest ocm cli version to install

Reply:
Added a hyperlink to the section "Install OCM-CLI" which has example to install latest ocm cli version